helm/tools/templates/deployer/deployerRole.yaml

```diff
@@ -0,0 +1,230 @@
+      - get
+      - list
+      - watch
+      - update
+  - apiGroups:
+      - image.openshift.io
+    resources:
+      - imagestreamtags
+    verbs:
+      - delete
+  - apiGroups:
+      - project.openshift.io
+      - ""
+    attributeRestrictions: null
+    resources:
+      - projects
+    verbs:
+      - get
+  - apiGroups:
+      - ""
+    attributeRestrictions: null
+    resources:
+      - replicationcontrollers
+      - persistentvolumeclaims
+      - services
+      - secrets
+      - configmaps
+      - endpoints
+      - pods
+      - pods/exec
+    verbs:
+      - watch
+      - list
+      - get
+      - create
+      - update
+      - patch
+      - delete
+      - deletecollection
+  - apiGroups:
+      - ""
+    attributeRestrictions: null
+    resources:
+      - pods/status
+      - pods/log
+    verbs:
+      - watch
+      - list
+      - get
+  - apiGroups:
+      - apps.openshift.io
+    attributeRestrictions: null
+    resources:
+      - deploymentconfigs
+    verbs:
+      - get
+      - create
+      - update
+      - patch
+  - apiGroups:
+      - apps
+    attributeRestrictions: null
+    resources:
+      - statefulsets
+    verbs:
+      - get
+      - create
+      - delete
+      - update
+      - patch
+  - apiGroups:
+      - route.openshift.io
+    attributeRestrictions: null
+    resources:
+      - routes
+    verbs:
+      - get
+      - create
+      - update
+      - patch
+      - delete
+  - apiGroups:
+      - template.openshift.io
+    attributeRestrictions: null
+    resources:
+      - processedtemplates
+    verbs:
+      - create
+  - apiGroups:
+      - route.openshift.io
+    attributeRestrictions: null
+    resources:
+      - routes
+    verbs:
+      - get
+      - create
+      - update
+      - patch
+  - apiGroups:
+      - route.openshift.io
+      - ""
+    attributeRestrictions: null
+    resources:
+      - routes/custom-host
+    verbs:
+      - create
+  - apiGroups:
+      - batch
+    attributeRestrictions: null
+    resources:
+      - jobs
+      - cronjobs
+    verbs:
+      - get
+      - create
+      - update
+      - patch
+      - delete
+      - watch
+      - list
+  - apiGroups:
+      - policy
+    resources:
+      - poddisruptionbudgets
+    verbs:
+      - get
+      - create
+      - update
+      - patch
+      - delete
+  - apiGroups:
+      - ""
+    resources:
+      - serviceaccounts
+    verbs:
+      - get
+      - list
+      - create
+      - update
+      - patch
+      - delete
+  - apiGroups:
+      - "rbac.authorization.k8s.io"
+      - "authorization.openshift.io"
+    resources:
+      - roles
+      - rolebindings   
+    verbs:
+      - get
+      - list
+      - create
+      - update
+      - patch
+      - delete
+  - apiGroups:
+      - "ops.gov.bc.ca"
+    resources:
+      - sysdig-teams
+    verbs:
+      - delete
+      - deletecollection
+      - get
+      - list
+      - patch
+      - create
+      - update
+      - watch
```

Comment:
This is probably the only role our service account needs to have right now, right? Granting our role permissions we're not going to use is a [Least Privilege](https://cheatsheetseries.owasp.org/cheatsheets/Authorization_Cheat_Sheet.html#enforce-least-privileges) violation. Can we try deleting the other roles and making sure our service account can still deploy?